### DISCUSSION TITLE
esmodule setting in readme is not clear - am I doing it right?

### DISCUSSION THREAD

**Discussion by cekvenich2** [parcel-bundler/parcel#6833] [Q&A]
Based on the readme here:
https://github.com/parcel-bundler/parcel

It says you can set package.json to esmodule.

But give no example of a proper format for outputFormat.

Am I doing it right here?
https://github.com/axthe/axthe/blob/main/static/package.json

**Comment by mischnic** [comment]
You should choose a different name for the target "browser", that is a target for libraries by default: https://v2.parceljs.org/features/targets/#library-targets

**Comment by cekvenich2** [comment]
@mischnic And how would I enable that target>

**Reply by mischnic** [reply]
As a CLI flag: `--target mytarget`

**Comment by cekvenich2** [comment]
I get ***@***.***/core: Only browser targets are supported in serve mode"
